test/units/module_utils/common/process/test_get_bin_path.py

```diff
@@ -15,9 +15,9 @@ def test_get_bin_path(mocker):
     mocker.patch.dict('os.environ', {'PATH': path})
     mocker.patch('os.pathsep', ':')
 
-    mocker.patch('os.path.exists', side_effect=[False, True])
     mocker.patch('os.path.isdir', return_value=False)
     mocker.patch('ansible.module_utils.common.process.is_executable', return_value=True)
+    mocker.patch('os.path.exists', side_effect=[False, True])
```

Comment:
I would add an explaining comment on why this needs to be last.

Reply:
Done.